src/core_atmosphere/physics/mpas_atmphys_driver_pbl.F

```diff
@@ -651,37 +823,112 @@ subroutine driver_pbl(itimestep,configs,mesh,sfc_input,diag_physics,tend_physics
+       call mpas_pool_get_config(configs,'config_mynn_edmf_output',bl_mynn_edmf_output)
+       call mpas_pool_get_config(configs,'config_mynn_closure'    ,bl_mynn_closure    )
+       call mpas_pool_get_config(configs,'config_mynn_mixscalars' ,bl_mynn_mixscalars )
+       call mpas_pool_get_config(configs,'config_mynn_mixclouds'  ,bl_mynn_cloudmix   )
+       call mpas_pool_get_config(configs,'config_mynn_mixqt'      ,bl_mynn_mixqt      )
+       call mpas_pool_get_config(configs,'config_mynn_tkeadvect'  ,bl_mynn_tkeadvect  )
+       call mpas_pool_get_config(configs,'config_mynn_tkebudget'  ,bl_mynn_tkebudget  )
+
+       call mpas_log_write(' ')
+       call mpas_log_write('--- enter subroutine mynn_bl_driver:')
+       call mpas_log_write('--- config_mynn_cloudpdf    = $i',intArgs=(/bl_mynn_cloudpdf/))
+       call mpas_log_write('--- config_mynn_mixlength   = $i',intArgs=(/bl_mynn_mixlength/))
+       call mpas_log_write('--- config_mynn_stfunc      = $i',intArgs=(/bl_mynn_stfunc/))
+       call mpas_log_write('--- config_mynn_topdown     = $i',intArgs=(/bl_mynn_topdown/))
+       call mpas_log_write('--- config_mynn_scaleaware  = $i',intArgs=(/bl_mynn_scaleaware/))
+       call mpas_log_write('--- config_mynn_dheat_opt   = $i',intArgs=(/bl_mynn_dheat_opt/))
+       call mpas_log_write('--- config_mynn_edmf        = $i',intArgs=(/bl_mynn_edmf/))
+       call mpas_log_write('--- config_mynn_edmf_dd     = $i',intArgs=(/bl_mynn_edmf_dd/))
+       call mpas_log_write('--- config_mynn_edmf_mom    = $i',intArgs=(/bl_mynn_edmf_mom/))
+       call mpas_log_write('--- config_mynn_edmf_tke    = $i',intArgs=(/bl_mynn_edmf_tke/))
+       call mpas_log_write('--- config_mynn_edmf_output = $i',intArgs=(/bl_mynn_edmf_output/))
+       call mpas_log_write('--- config_mynn_mixscalars  = $i',intArgs=(/bl_mynn_mixscalars/))
+       call mpas_log_write('--- config_mynn_mixclouds   = $i',intArgs=(/bl_mynn_cloudmix/))
+       call mpas_log_write('--- config_mynn_mixqt       = $i',intArgs=(/bl_mynn_mixqt/))
+       call mpas_log_write('--- config_mynn_tkeadvect   = $l',logicArgs=(/bl_mynn_tkeadvect/))
+       call mpas_log_write('--- config_mynn_tkebudget   = $i',intArgs=(/bl_mynn_tkebudget/))
+       call mpas_log_write('--- config_mynn_closure     = $r',realArgs=(/bl_mynn_closure/))
+       call mpas_log_write(' ')
+       call mpas_log_write('--- f_qc   = $l',logicArgs=(/f_qc/)  )
+       call mpas_log_write('--- f_qi   = $l',logicArgs=(/f_qi/)  )
+       call mpas_log_write('--- f_qs   = $l',logicArgs=(/f_qs/)  )
+       call mpas_log_write('--- f_qoz  = $l',logicArgs=(/f_qoz/) )
+       call mpas_log_write('--- f_nc   = $l',logicArgs=(/f_nc/)  )
+       call mpas_log_write('--- f_ni   = $l',logicArgs=(/f_ni/)  )
+       call mpas_log_write('--- f_nifa = $l',logicArgs=(/f_nifa/))
+       call mpas_log_write('--- f_nwfa = $l',logicArgs=(/f_nwfa/))
+       call mpas_log_write('--- f_nbca = $l',logicArgs=(/f_nbca/))
```

Comment:
I'd propose removing these print statements, as they show up for every time step in the log file.

Reply:
As requested, I commented out the extra print statements. Although they add a little bit of clutter around the call to subroutine mynn_bl_driver, I thought that at least initially, it was important for users to know exactly which options are being used in the MYNN PBL scheme. Defaults values for the different options are of course in Registry.xml.